docker-bench-security.sh

```diff
@@ -103,11 +132,15 @@ fi
 totalChecks=0
 currentScore=0
 
-logit "Initializing $(date)\n"
+logit "Initializing $(date +%Y-%m-%dT%H:%M:%S%:z)\n"
+appendjson
 beginjson "$version" "$(date +%s)"
 
 # Load all the tests from tests/ and run them
 main () {
+  logit "\n${bldylw}Section A - Checks result${txtrsr}"
```

Comment:
Nitpicking - "Check results"? Dont know which one is more correct english.

Reply:
It's done